content/docs/cli.md

```diff
@@ -21,18 +21,20 @@ brew tap kudobuilder/tap
 brew install kudo-cli
 ```
 
-or you can compile and install the plugin from your `$GOPATH/src/github.com/kudobuilder/kudo` root folder via:
+Another alternative is `krew` the package manager for kubectl plugins [doc](https://github.com/kubernetes-sigs/krew)
 
 ```bash
-make cli-install
+kubectl krew install kudo
 ```
 
-Another alternative is `krew` the package manager for kubectl plugins [doc](https://github.com/kubernetes-sigs/krew)
+or you can compile and install the plugin from your `$GOPATH/src/github.com/kudobuilder/kudo` root folder via:
 
 ```bash
-kubectl krew install kudo
+make cli-install
 ```
 
+**note:** Building the CLI with `make` will work for all functions with the exception of `kudo init`.  Building and running kudo this way is for developers and advanced users.
```

Comment:
I agree it's for advanced users, but in this case we should not be mentioning it here, in the "quick start for newbies" doc.

Putting the warning after the command means that some users will first try it out, and then when it seemingly worked they will read the warning and say "meh".

Also, can you explain why `kudo init` will not work? I think it well might, but the problem is with using unreleased versions of KUDO.

Reply:
@porridge I think the note should be here as long as we have the `make ...` command as an option here... are you suggesting a move?   I'll take a look if there is a logic place for it.

Reply:
I'm suggesting that both the `make ...` and the note should be removed from this page. Developers and advanced users who want to use the bleeding edge `master` already know how to clone a github repo and read its README.

OTOH, what we seem to be missing here is the simple option to just download the CLI plugin from the asset list of https://github.com/kudobuilder/kudo/releases/latest (latest stable release), `chmod +x` it and put it in the `$PATH`.

Reply:
I completely agree with you on this...   dev details in project repo... this site is for users only!